### DISCUSSION TITLE
How to change min and max values on a pole figure.

### DISCUSSION THREAD

**Discussion by MWalag** [mtex-toolbox/mtex#1390] [Ask Anything]
Hello!
Is it possible to change values of min and max on attached pole figure? I think, when the minimum will be set to lower value the orientation marked on a south pole will be visible as isoline.
![Przechwytywanie](https://user-images.githubusercontent.com/99949071/160281754-052ed3ba-5390-4a62-a3a7-53fc3115a1c6.PNG)
I'm using default plotting comands:
Firstly I import uxd pole figure data with defocusing and then:
odf=calcODF(pf);
[ori, vol] = calcComponents(odf);
ori
vol*100
plotPDF(odf,pf.h,'contour','linewidth',1.5,'antipodal','projection','eangle','minmax')
annotate(ori)

**Comment by kilir** [comment]
Hi,
the option 'contours' allows you to specify the levels of the contours, e.g. `plotPDF(odf,pf.h,'contour','contours',[0:1:3])`. However contours close to 0 are usually not very helpful I'd say.

Cheers,
Rüdiger
